src/downloadMTD.js

```diff
@@ -12,9 +14,10 @@ const metaUpdate = require('./metaUpdate')
 const create = require('reactive-storage').create
 const Immutable = require('immutable')
 
-module.exports = (ob, fd) => {
+module.exports = (ob, fd, options) => {
   const offsets = create(Immutable.List([]))
   const loadedMETA = metaLoad(fd)
+    .map(meta => _.assign({}, options, meta))
```

Comment:
maybe ... I just copied what you wrote in the issue comment :-)

However, this still doesn't fix the issue